includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -1155,6 +1188,32 @@ public static function print_status_meta_box( $post ) {
 						?>
 						<?php self::display_invalid_url_validation_error_counts_summary( $post ); ?>
 					</div>
+
+					<div class="misc-pub-section">
+						<?php
+						$view_label     = __( 'View URL', 'amp' );
+						$queried_object = get_post_meta( $post->ID, '_amp_queried_object', true );
```

Comment:
Great idea to store the queried object. This will also help with displaying the title, like on [this design](https://sketch.cloud/s/xQRzx/all/page-1/amp-validation-errors-for-amp-test-embeds):

<img width="662" alt="will-help-with-title" src="https://user-images.githubusercontent.com/4063887/45459978-a1d70700-b6c0-11e8-8731-7dc63c5eb006.png">